jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -388,6 +388,8 @@ public void removeVirtualHosts(String... virtualHosts)
     @ManagedAttribute(value = "Virtual hosts accepted by the context", readonly = true)
     public List<String> getVirtualHosts()
     {
+        if (_vhosts == null)
```

Comment:
Null-checking a final field? This indicates a constructor escape, that's not a good sign.

Reply:
This happens during toString() from logging.

```
SLF4J: Failed toString() invocation on an object of type [org.eclipse.jetty.server.handler.ContextHandler]
Reported exception:
java.lang.NullPointerException: Cannot invoke "java.util.List.stream()" because "this._vhosts" is null
	at org.eclipse.jetty.server@12.0.6-SNAPSHOT/org.eclipse.jetty.server.handler.ContextHandler.getVirtualHosts(ContextHandler.java:391)
	at org.eclipse.jetty.server@12.0.6-SNAPSHOT/org.eclipse.jetty.server.handler.ContextHandler.toString(ContextHandler.java:1032)
	at org.slf4j@2.0.9/org.slf4j.helpers.MessageFormatter.safeObjectAppend(MessageFormatter.java:291)
	at org.slf4j@2.0.9/org.slf4j.helpers.MessageFormatter.deeplyAppendParameter(MessageFormatter.java:263)
	at org.slf4j@2.0.9/org.slf4j.helpers.MessageFormatter.arrayFormat(MessageFormatter.java:225)
	at org.slf4j@2.0.9/org.slf4j.helpers.MessageFormatter.arrayFormat(MessageFormatter.java:160)
	at org.eclipse.jetty.logging@12.0.6-SNAPSHOT/org.eclipse.jetty.logging.StdErrAppender.format(StdErrAppender.java:162)
	at org.eclipse.jetty.logging@12.0.6-SNAPSHOT/org.eclipse.jetty.logging.StdErrAppender.emit(StdErrAppender.java:93)
	at org.eclipse.jetty.logging@12.0.6-SNAPSHOT/org.eclipse.jetty.logging.JettyLogger.emit(JettyLogger.java:633)
	at org.eclipse.jetty.logging@12.0.6-SNAPSHOT/org.eclipse.jetty.logging.JettyLogger.debug(JettyLogger.java:172)
	at org.eclipse.jetty.util@12.0.6-SNAPSHOT/org.eclipse.jetty.util.component.ContainerLifeCycle.addBean(ContainerLifeCycle.java:419)
	at org.eclipse.jetty.util@12.0.6-SNAPSHOT/org.eclipse.jetty.util.component.ContainerLifeCycle.addBean(ContainerLifeCycle.java:317)
	at org.eclipse.jetty.server@12.0.6-SNAPSHOT/org.eclipse.jetty.server.Handler$Wrapper.<init>(Handler.java:696)
	at org.eclipse.jetty.server@12.0.6-SNAPSHOT/org.eclipse.jetty.server.Handler$Wrapper.<init>(Handler.java:682)
	at org.eclipse.jetty.server@12.0.6-SNAPSHOT/org.eclipse.jetty.server.handler.ContextHandler.<init>(ContextHandler.java:177)
	at org.eclipse.jetty.server@12.0.6-SNAPSHOT/org.eclipse.jetty.server.handler.ContextHandler.<init>(ContextHandler.java:167)
	at org.eclipse.jetty.server@12.0.6-SNAPSHOT/org.eclipse.jetty.server.LoggerDebugTest.testWrapperNPE(LoggerDebugTest.java:69)
```